tests/record_test.py

```diff
@@ -378,8 +377,8 @@ def test_transform_without_update_returns_same_precord():
 
 
 class Application(PRecord):
-    name = field(type=(six.text_type,) + six.string_types)
-    image = field(type=(six.text_type,) + six.string_types)
+    name = field(type=(str, str))
```

Comment:
No point in having double strings here, right?

Reply:
I did not really got it, but when I did execute :

`(six.text_type,) + six.string_types`

It returned `(str, str)`